### ISSUE TITLE
Recover button (in FeinCMS) raises SQL error

### ISSUE THREAD

**Issue by djowett** [etianen/django-reversion#689]
```
2017-12-13 14:33:14,148: manage[2482]: ERROR: Internal Server Error: /cms/blogpost/recover/
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/django/core/handlers/base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/lib/python2.7/dist-packages/django/core/handlers/base.py", line 147, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/lib/python2.7/dist-packages/django/utils/decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/django/views/decorators/cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/django/contrib/admin/sites.py", line 244, in inner
    return view(request, *args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/reversion/admin.py", line 279, in recoverlist_view
    context,
  File "/usr/lib/python2.7/dist-packages/django/shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "/usr/lib/python2.7/dist-packages/django/template/loader.py", line 97, in render_to_string
    return template.render(context, request)
  File "/usr/lib/python2.7/dist-packages/django/template/backends/django.py", line 95, in render
    return self.template.render(context)
  File "/usr/lib/python2.7/dist-packages/django/template/base.py", line 206, in render
    return self._render(context)
  File "/usr/lib/python2.7/dist-packages/django/template/base.py", line 197, in _render
    return self.nodelist.render(context)
  File "/usr/lib/python2.7/dist-packages/django/template/base.py", line 992, in render
    bit = node.render_annotated(context)
  File "/usr/lib/python2.7/dist-packages/django/template/base.py", line 959, in render_annotated
    return self.render(context)
  File "/usr/lib/python2.7/dist-packages/django/template/loader_tags.py", line 173, in render
    return compiled_parent._render(context)
  File "/usr/lib/python2.7/dist-packages/django/template/base.py", line 197, in _render
    return self.nodelist.render(context)
  File "/usr/lib/python2.7/dist-packages/django/template/base.py", line 992, in render
    bit = node.render_annotated(context)
  File "/usr/lib/python2.7/dist-packages/django/template/base.py", line 959, in render_annotated
    return self.render(context)
  File "/usr/lib/python2.7/dist-packages/django/template/loader_tags.py", line 173, in render
    return compiled_parent._render(context)
  File "/usr/lib/python2.7/dist-packages/django/template/base.py", line 197, in _render
    return self.nodelist.render(context)
  File "/usr/lib/python2.7/dist-packages/django/template/base.py", line 992, in render
    bit = node.render_annotated(context)
  File "/usr/lib/python2.7/dist-packages/django/template/base.py", line 959, in render_annotated
    return self.render(context)
  File "/usr/lib/python2.7/dist-packages/django/template/loader_tags.py", line 69, in render
    result = block.nodelist.render(context)
  File "/usr/lib/python2.7/dist-packages/django/template/base.py", line 992, in render
    bit = node.render_annotated(context)
  File "/usr/lib/python2.7/dist-packages/django/template/base.py", line 959, in render_annotated
    return self.render(context)
  File "/usr/lib/python2.7/dist-packages/django/template/defaulttags.py", line 325, in render
    if match:
  File "/usr/lib/python2.7/dist-packages/django/db/models/query.py", line 266, in __nonzero__
    return type(self).__bool__(self)
  File "/usr/lib/python2.7/dist-packages/django/db/models/query.py", line 262, in __bool__
    self._fetch_all()
  File "/usr/lib/python2.7/dist-packages/django/db/models/query.py", line 1074, in _fetch_all
    self._result_cache = list(self.iterator())
  File "/usr/lib/python2.7/dist-packages/django/db/models/query.py", line 52, in __iter__
    results = compiler.execute_sql()
  File "/usr/lib/python2.7/dist-packages/django/db/models/sql/compiler.py", line 848, in execute_sql
    cursor.execute(sql, params)
  File "/usr/lib/python2.7/dist-packages/django/db/backends/utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "/usr/lib/python2.7/dist-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/usr/lib/python2.7/dist-packages/django/db/utils.py", line 95, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/usr/lib/python2.7/dist-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/usr/lib/python2.7/dist-packages/django/db/backends/mysql/base.py", line 112, in execute
    return self.cursor.execute(query, args)
  File "/usr/lib/python2.7/dist-packages/MySQLdb/cursors.py", line 226, in execute
    self.errorhandler(self, exc, value)
  File "/usr/lib/python2.7/dist-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorvalue
ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'varchar(191))\n                WHERE\n                    V.`db` = 'default' AND\n ' at line 4")
```

I'm suspecting bb4d193bc may have something to do with it because I see the error on 2.0.11, but not 2.0.10

**Comment by etianen** [comment]
This is already fixed in 2.0.12.

**Comment by djowett** [comment]
Ok, thanks looks good in 2.0.12 to me.
